### ISSUE TITLE
Questions About the model

### ISSUE THREAD

**Issue by ShiinaMitsuki** [louis-she/center-loss.pytorch#3]
Hi, thanks for your pytorch implementation of the center face .
I played with your code for a while, and I may found some problem.

1. lfw dataset, It seems that you use the whole picture instead of using the face detection bounding boxes or just using center cropped images for training and testing,  take a look at some pictures in the dataset, 

![jiang_zemin_0001](https://user-images.githubusercontent.com/16696202/46643966-eca94900-cbb0-11e8-8c86-3558887c3ceb.jpg)

![jiang_zemin_0005](https://user-images.githubusercontent.com/16696202/46643989-034fa000-cbb1-11e8-8091-fb26be33a783.jpg)

by this way, too much background are envoled.

2. overlap between training set and test set. 
```
    for klass, name in enumerate(names):
        def add_class(image):
            image_path = os.path.join(images_root, name, image)
            return (image_path, klass, name)

        images_of_person = os.listdir(os.path.join(images_root, name))
        total = len(images_of_person)

        training_set += map(
                add_class,
                images_of_person[:ceil(total * train_val_split)])
        validation_set += map(
                add_class,
                images_of_person[floor(total * train_val_split):])
```
I think you should use samples listed in the _pairsDevTrain.txt_ to train and _pairsDevTest.txt_ to test by the [lfw paper](http://vis-www.cs.umass.edu/lfw/lfw.pdf)

**Comment by louis-she** [comment]
will look into it, Thanks!
